default/scripting/common/priorities.macros

```diff
@@ -15,5 +36,17 @@ LATE_PRIORITY
 VERY_LATE_PRIORITY
 '''120'''
 
-LATEST_GROWTH_PRIORITY
-'''99999'''
+AFTER_ALL_TARGET_MAX_METERS_PRIORITY
+'''1000'''
+
+
+FOCUS_CHANGE_PENALTY_PRIORITY
+'''1100'''
+
+
+METER_OVERRIDE_PRIORITY
+'''2000'''
+
+
+END_CLEANUP_PRIORITY
+'''5000'''
```

Comment:
EOL

Reply:
not sure why / if having endlines matters...

Reply:
Some of the parsing has issue with not reading/matching last line in file when it does not end in newline.